### PR TITLE
Added Serverless & Container Aware Architecture (SCAR)

### PR DESCRIPTION
**Pull request by vegasbrianc**
Added [L 388] https://github.com/grycap/scar

**Serverless Container-aware Architectures (SCAR) is a serverless framework that allows easy deployment and execution of containers (e.g. Docker) in Serverless environments (e.g. Lambda)**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)
I read it again, I promise
- The project submitted is conform to the quality standards outlined in the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)
Yes
- Go to the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md) and read it again.
Yes, I read it again

RULES (because you probably didn't read the guidelines): 
* ORDER: The entries are in lexical order (English letter order) 
* CREDIT: Each entry must contains the name of the author with a link to its github account!
* LINK: the link should be to the source code (https://github/account/project) not the website (https://myproject.io <- no!)
* MONEY: indicates if one needs a paying subscription to use that project you could just add ($$$) next to the name ([my project - $$$] (link to my project) )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -385,6 +385,7 @@ Services to securely store your Docker images.
 * [FaaS](https://github.com/alexellis/faas) - Docker Serverless/Functions as a Service (on Docker Swarm). By [@alexellis](https://github.com/alexellis/)
 * [Funker](https://github.com/bfirsh/funker-example-voting-app) - Functions as Docker containers example voting app. By [@bfirsh](https://github.com/bfirsh)
 * [IronFunctions](https://github.com/iron-io/functions) - The serverless microservices platform FaaS (Funcitons as a Service) which uses Docker containers to run Any language or AWS Lambda functions
+* [SCAR](https://github.com/grycap/scar) - Serverless & Container Aware Architectures (SCAR) by [Grycap](https://github.com/grycap)
```
Comment (devhkr) [inline README.md]: A bit more description would be nice, like as you did on the PR comment.

`Serverless & Container Aware Architecture (SCAR) is a serverless framework that allows easy deployment and execution of containers in Serverless environments i.e. Lambda`

And `@` symbol is missing on the author's GitHub link.

**Comment by vegasbrianc** [comment]
@devhkr Updated the PR with your feedback. Thanks!

**Comment by vegasbrianc** [comment]
Hi @devhkr OK, I think it is right now. Thanks for the help.

File: README.md [hunk 2]
```diff
@@ -385,6 +385,7 @@ Services to securely store your Docker images.
 * [FaaS](https://github.com/alexellis/faas) - Docker Serverless/Functions as a Service (on Docker Swarm). By [@alexellis](https://github.com/alexellis/)
 * [Funker](https://github.com/bfirsh/funker-example-voting-app) - Functions as Docker containers example voting app. By [@bfirsh](https://github.com/bfirsh)
 * [IronFunctions](https://github.com/iron-io/functions) - The serverless microservices platform FaaS (Funcitons as a Service) which uses Docker containers to run Any language or AWS Lambda functions
+* [SCAR](https://github.com/grycap/scar) - Serverless Container-aware Architectures (SCAR) is a serverless framework that allows easy deployment and execution of containers (e.g. Docker) in Serverless environments (e.g. Lambda) (SCAR) by [@grycap](https://github.com/grycap)
```
Comment (devhkr) [inline README.md]: LGTM now, just quick question: is it on purpose that the acronym `(SCAR)` is mentioned twice in the description? 

[...] Serverless Container-aware Architectures (SCAR) [...]
[...] erverless environments (e.g. Lambda) (SCAR) by [...]

Comment (vegasbrianc) [inline README.md]: DOH! Fixing now

Comment (devhkr) [inline README.md]: Sorry @vegasbrianc

Don't if it is the heat which let me work over 100% to stay focused 😉

Comment (vegasbrianc) [inline README.md]: @devhkr I also blame it on the heat and lack of AC...HAHA Or could be I'm just out of practice as I've been away from the keyboard for so long.

**Comment by devhkr** [comment]
Anyway, thanks for sharing @vegasbrianc and thanks a lot for your patience.

**Pull request merged**